public/locales/en/translation.json

```diff
@@ -38,11 +38,12 @@
             "validated": "This scenario is validated, you cannot edit its parameters",
             "noTabs": "No parameters to edit"
           },
-          "discard": "Discard modifications",
+          "discard": "DISCARD",
           "edit": "Edit parameters",
-          "launch": "Launch scenario",
+          "launch": "LAUNCH",
           "update": {
-            "launch": "Update and launch scenario"
+            "update": "SAVE",
+            "launch": "SAVE AND LAUNCH"
```

Comment:
Do we still have the button update and launch ? If it's necessary for older versions, why change the name ?